test/integration/ComputeEngineIntegrationTest.java

```diff
@@ -35,17 +38,17 @@ public class ComputeEngineIntegrationTest{
 	public void test() {
 		
 		int[] arr = {1, 10, 25};
-		User user = new User();
+		IntegrationUser user = new IntegrationUser(new ComputeEngine());//put in the tested component
 		TestUserDataInput dataIn = new TestUserDataInput(arr, '\n'); // converts to a list<Integer> in the constructor
 		
 		UserDataStream output = (UserDataStream) user.sendUserRequest(dataIn);
 		
 		String realOutput = output.toString();
-		String expectedOutput = "1\n10\n25\n";
+		String expectedOutput = "1.0\n0.38500008\n0.35360003\n";
```

Comment:
Glad these are the expected values, but I worry about floating point precision. Maybe we check up to a certain decimal place?